docs/Common/Enums/index.md

```diff
@@ -1305,10 +1308,25 @@
 |CLOUD|1||
 |BOTH|2||
 
+<<<<<<< HEAD
```

Comment:
```suggestion

```

Reply:
@ShobhitAd fixed in https://github.com/smartdevicelink/sdl_hmi_integration_guidelines/pull/214/commits/d20c6eb18a1a17d0b0da563deb346b018ff6fdd5